src/EditorFeatures/Test2/FindReferences/FindReferencesTests.AccessorSymbols.vb

```diff
@@ -0,0 +1,1685 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Threading.Tasks
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.FindReferences
+    Partial Public Class FindReferencesTests
```

Comment:
>FindReferencesTests [](start = 25, length = 19)

Could you also test anonymous objects? `new { A = 1 }`

Reply:
Also, it seems you implemented the logic for cref in xml doc, but I didn't find a corresponding test. Thanks

---
In reply to: [206351014](https://github.com/dotnet/roslyn/pull/28231#discussion_r206351014) [](ancestors = 206351014)

Reply:
> Could you also test anonymous objects? var o = new { A = 1 }, then use o.A and FAR on both locations of A.

Anonymous types are already heavily tested in: FindReferencesTests.AnonymousTypeSymbols.vb